### DISCUSSION TITLE
Quantum hardware and runtime

### DISCUSSION THREAD

**Discussion by AlexanderGeng** [qiskit-community/prototype-quantum-kernel-training#15] [Q&A] [answered]
Hey,

first of all thanks for the wonderful repository. It is quite informative.
I saw @jenglick changed the ReadMe and deleted the Using Quantum Service part and the QuantumKernelTrainerClient.

Are these two things no longer the focus of this repository?
I would be very interested in them, especially what to expect on real hardware using qiskit runtime.

**Comment by jenglick** [comment, marked as answer]
Hi Alex! Thanks for reaching out. A runtime program for the QKT toolkit is not currently available. (Note that quantum kernel training with QKT itself can be run on quantum hardware -- just not via a runtime program at the moment)

However, there is a separate runtime program that is already available for testing a specific case of kernel training called Quantum Kernel Alignment on quantum hardware or simulator. You can find more information about [this program](https://quantum-computing.ibm.com/services?program=quantum-kernel-alignment) called `quantum-kernel-alignment` on the IBM Quantum Services webpage.

**Reply by AlexanderGeng** [reply]
Ok perfect, I see.
Thanks for the quick answer

**Comment by vignanreddy2907** [comment]
What is advantage of Quantum Machine Learning over Classical ML, is it accuracy or computation time.
i mean in what cases and scenarios(data sets) it is better use QML instead of Classical ML?
when i performed classification of iris data using SVC and QSVC, SVC gave more accuracy in less time compare to QSVC on ibmq_qasm_simulator.
